firewood/src/hashednode.rs

```diff
@@ -81,15 +81,15 @@ impl<T: ReadLinearStore> HashedNodeStore<T> {
     pub fn root_hash(&self) -> Result<TrieHash, Error> {
         debug_assert!(self.modified.is_empty());
         if let Some(addr) = self.nodestore.root_address() {
-            #[cfg(feature = "nightly")]
+            #[cfg(nightly)]
```

Comment:
Don't know why linting now fails without this change @rkuris ?

Reply:
Note that this fails the stable clippy, since it checks for the invalid "nightly" cfg. Perhaps not a huge deal but we'll have to fix this again later probably.